### PR TITLE
Update maxes

### PR DESCRIPTION
**Pull request by madisonrogers**
The admin can now update the athletes maxes from their profile page.

### CODE REVIEW

File: api/controllers/userCtrl.js [hunk 1]
```diff
@@ -606,6 +606,23 @@ module.exports.getAllExercises = function(req, res) {
 }
 
 
+module.exports.updateMaxs = function(req, res) {
```
Comment (rileymiller) [inline api/controllers/userCtrl.js]: change this to updateMaxes

File: views/athlete.pug [hunk 2]
```diff
@@ -2,26 +2,73 @@ extends layout
...
+                             i.far.fa-edit
+    
 mixin table-condensed(header,data)
-	+table(header,data,"table-condensed")
+    +table(header,data,"table-condensed")
+append scripts
+    script(src="https://d3js.org/d3.v3.min.js")
+    script(src='/js/athleteViewHistoryCtrl.js')
+    style(type='text/css').
+        .line {
```
Comment (rileymiller) [inline views/athlete.pug]: Could you move these styles into the SASS file?

File: api/controllers/userCtrl.js (hunk 1 above)
Comment (madisonrogers) [reply api/controllers/userCtrl.js]: I changed this in another PR, sorry I didn't see this comment

File: views/athlete.pug (hunk 2 above)
Comment (madisonrogers) [reply views/athlete.pug]: I put them here so they do not conflict with the createPhase styles

**Pull request merged**